### DISCUSSION TITLE
Cosmo SkyMed Processing

### DISCUSSION THREAD

**Discussion by zsmith57** [gmtsar/gmtsar#516] [General]
Hello,

I installed GMTSAR and have processed Sentinel-1 and ALOS-1 data and everything worked fine. I then attempted to run the Cosmo SkyMed examples that are available on the GMTSAR website and I get errors. I used the following prompts to generate the config file and initiate processing (which come from the readme file):

pop_config.csh CSK_RAW > config.txt
p2p_processing.csh CSK_RAW CSKS2_RAW_B_HI_10_HH_RD_SF_20140105040332_20140105040340 CSKS2_RAW_B_HI_10_HH_RD_SF_20140121040327_20140121040334 config.txt

I then got an error. The first several lines of the error message are:

make_raw_csk: error while loading shared libraries: libhdf5.so.103: cannot open shared object file: No such file or directory
make_raw
[ErrorMessage.txt](https://github.com/gmtsar/gmtsar/files/9883564/ErrorMessage.txt)
[ErrorMessage.txt](https://github.com/gmtsar/gmtsar/files/9883566/ErrorMessage.txt)
_csk: error while loading shared libraries: libhdf5.so.103: cannot open shared object file: No such file or directory

I can provide the full error message if needed. It appears that it is unable to find files in the directory. This is not an issue when processing Sentinel-1 or ALOS-1, so I am not sure what is different about the Cosmo SkyMed processing. I'm not sure if there are additional installation steps I need to take to be able to process Cosmo SkyMed data. Any insights on this would be appreciated.

Thank you,
Zachary

**Comment by Xiaohua-Eric-Xu** [comment]
Feels like it's your hdf5 library that is broken. You could try re-installing your hdf5 lib, then recompile `GMT` and `GMTSAR`, then it hopefully will work.

**Reply by zsmith57** [reply]
Thanks Xiaohua for your fast response. I will uninstall everything and try installing again.
